docs/conf.py

```diff
@@ -13,6 +13,8 @@
 
 import sys, os
 
+sys.path.insert(0, os.path.dirname(os.path.dirname(__file__)))
```

Comment:
I want to include the root project directory, so I can use `import mosql`. Previously you need to always `make -C docs html` to create docs, but I want to make this work with `cd docs && make html` because there’s no `make -C` on Windows.

~~~python
__file__ -> <project_root>/docs/conf.py
os.path.dirname(__file__) -> <project_root>/docs/
os.path.dirname(os.path.dirname(__file__)) -> <project_root>/
~~~